src/main/java/juke/commands/JukeFindTaskCommand.java

```diff
@@ -0,0 +1,63 @@
+package juke.commands;
+
+import java.util.List;
+
+import juke.Juke;
+import juke.commons.utils.StringUtils;
+import juke.responses.Response;
+import juke.tasks.JukeTask;
+import juke.tasks.TaskList;
+
+/**
+ * Action that finds a Task in the {@code TaskList} according to the
+ * task description.
+ */
+public class JukeFindTaskCommand extends JukeCommand {
+    /** {@code TaskList} to manage all tasks. */
+    private final TaskList taskList;
+
+    /** Word of interest to search. */
+    private final String word;
+
+    /**
+     * Creates an instance of {@code JukeFindTaskCommand}.
+     *
+     * @param taskList {@code TaskList} instance
+     * @param word Word of interest to search
+     */
+    public JukeFindTaskCommand(TaskList taskList, String word) {
+        this.taskList = taskList;
+        this.word = word;
+    }
+
+    /**
+     * Carries out an action when the command is executed.
+     *
+     * @param response {@code Response} object that contains response from Juke and the user
+     * @return {@code Response} object that contains response from Juke and the user
+     */
+    @Override
+    public Response execute(Response response) {
+        List<JukeTask> foundTask = this.taskList.findTask(this.word);
```

Comment:
Perhaps this can be named foundTasks instead?

Reply:
Oh right, another oversight on my part! Thank you for bringing this up!